src/xmpp/omemo.c

```diff
@@ -462,6 +462,10 @@ _omemo_receive_devicelist(xmpp_stanza_t *const stanza, void *const userdata)
             }
         }
     }
+    if (item && g_strcmp0(xmpp_stanza_get_id(item), "current") != 0 ) {
```

Comment:
Please put this in the else block above so strcmp() doesnt need to run twice :-)